src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -11,6 +11,7 @@
 from ._router import Router
 from ._task import Task
 from ._ui_router import UIRouter
+from .auth_classes import BaseAuth
```

Comment:
BTW, since the `auth` service is becoming overcrowded, it's ok to split it if smaller files or even a separate dir (I was in fact thinking on having per-backend files in order to support future dynamic loading of auth backends). Something like `services/auth/*` could work, but that would force you to update a few imports over there.

Reply:
I went with your suggestion and separated the auth classes in respective files in `services/auth/*`